package/models.py

```diff
@@ -67,7 +68,16 @@ class Package(BaseModel):
     documentation_url = models.URLField(_("Documentation URL"), blank=True, null=True, default="")
 
     commit_list = models.TextField(_("Commit List"), blank=True)
-
+    date_deprecated = models.DateTimeField(blank=True, null=True)
+    deprecated_by = models.ForeignKey(User, blank=True, null=True, related_name="deprecator", on_delete=models.PROTECT)
```

Comment:
This is a great addition. Are you integrating it anywhere?

Reply:
Follow-up: At a minimum, we should add this field to our raw_id_fields list to avoid the admin getting overwhelmed by 15k users.